filesystem/templatecopy.go

```diff
@@ -153,7 +163,14 @@ func writeTemplateToFile(tpl string, config interface{}, writepath string,
 		openingDelimiter = "{{"
 		closingDelimiter = "}}"
 	}
-	packageTemplate, err := template.New("").Delims(openingDelimiter, closingDelimiter).Funcs(sprig.TxtFuncMap()).Parse(tpl)
+	packageTemplate := template.New("")
+	var err error
+	methodMap := template.FuncMap{
+		"execTpl":        execTpl(packageTemplate),
+		"aesCbcPbkdfEnc": common.AesCbcEncryptWithPbkdfWrapper,
```

Comment:
Probably will help to have a prefix like say encryptAesCbcPbkdf and encrypt_ RsaCert, so that it is easier to decipher the various functions.